src/main/java/org/humancellatlas/ingest/core/MetadataDocument.java

```diff
@@ -60,24 +55,6 @@ protected MetadataDocument(EntityType type,
         this.validationState = ValidationState.DRAFT;
     }
 
-    public MetadataDocument addToSubmissionEnvelope(SubmissionEnvelope submissionEnvelope) {
-        SubmissionEnvelope openSubmission = this.getOpenSubmissionEnvelope();
-        if( openSubmission == null ){
-            if(this.validationState != ValidationState.DRAFT){
-                this.enactStateTransition(ValidationState.DRAFT);
-            }
-            this.submissionEnvelopes.add(submissionEnvelope);
-        }
-        else if (!openSubmission.getId().equals(submissionEnvelope.getId())){
-            String errorMessage = String.format("The %s metadata %s is still linked to a %s submission envelope %s.",
-                    this.getType(), this.getId(), openSubmission.getSubmissionState(), openSubmission.getId());
-            getLog().error(errorMessage);
-
-            throw new LinkToNewSubmissionNotAllowedException(errorMessage);
-        }
-        return this;
-    }
-
     public static List<ValidationState> allowedStateTransitions(ValidationState fromState) {
```

Comment:
Just to clarify, with this change it doesn't make sense to have this method in MetadataDocument, but should move to Project instead (if we intend to maintain the list of submissionEnvs it contains there).